staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -210,6 +234,10 @@ func (o *AuditLogOptions) AddFlags(fs *pflag.FlagSet) {
 		"Format of saved audits. \"legacy\" indicates 1-line text format for each event."+
 			" \"json\" indicates structured json format. Requires the 'AdvancedAuditing' feature"+
 			" gate. Known formats are "+strings.Join(pluginlog.AllowedFormats, ",")+".")
+	fs.StringVar(&o.Mode, "audit-log-mode", o.Mode,
```

Comment:
Extract to a separate function `AddBufferedFlags`

Reply:
I think you may think buffere backend has many modes. Here firstly from upper level audit is divided to log and webhook. And then lower each is divided modeBlocking and modeBatch(buffered).

Reply:
> I think you may think buffere backend has many modes

It will have more than one flag

> Here firstly from upper level audit is divided to log and webhook. And then lower each is divided modeBlocking and modeBatch(buffered).

...and this leads to code duplication that will grow even more once we make more things in the buffered backend configurable (batch size, buffer size, etc.)

Reply:
>...and this leads to code duplication that will grow even more once we make more things in the buffered backend configurable (batch size, buffer size, etc.)
You mean to make these configurable in this pr ?

Reply:
@sttts I do not see the need to extract bufferedBackend options now.  After all it is just a wrap of real backend.
And the current default values are not decided how to configure yet.
>	// TODO(ericchiang): Make these value configurable. Maybe through a
	// kubeconfig extension?